sample/www/jxcore/test/thaliscenarios.js

```diff
@@ -0,0 +1,79 @@
+var test = require('tape');
+var ThaliEmitter = require('../thali/thaliemitter');
+
+test('ThaliEmitter can call startBroadcasting and endBroadcasting without error', function (t) {
+  var e = new ThaliEmitter();
+
+  console.log('about to call startBroadcasting');
+  e.startBroadcasting((+ new Date()).toString(), 5000, function (err1) {
+    t.notOk(err1);
```

Comment:
Shouldn't we use plan(2) so we can make sure that both call backs actually happened? Right now if via some strange bug the startBroadcasting never actually called back we wouldn't realize there was an error.